src/config/rounds/round1.tsx

```diff
@@ -20,11 +24,16 @@ export const round1: RoundDescription<Round1ActionId> = {
       needs you to prove that you can deliver a working …
     </p>
   ),
-  effect: () => ({
-    capacityChange: 10,
-    userStoryChance: 30,
-    title: EFFECT_HIDDEN,
-  }),
+  effect: () => [
+    {
+      title: EFFECT_HIDDEN,
+      capacityChange: START_CAPACITY,
+    },
+    {
+      title: EFFECT_HIDDEN,
+      userStoryChance: START_USER_STORY_CHANCE,
+    },
+  ],
```

Comment:
rounds and gremlins can now return an array of effects. this is because having effects with multiple causes (capacity + userStoryChance) might cause bugs when filtering them